### PR TITLE
Update color for Sway

### PR DESCRIPTION
**Pull request by sdankel**
Changing the color for the Sway language to Fuel Labs' green.

image

## Checklist:

- [x] **I am changing the color associated with a language**
  
  - [x] I have obtained agreement from the wider language community on this color change.
    - Public discussion: https://forum.fuel.network/t/color-for-sway-language-in-github/3420
    - Brand guide: https://fuel-brand-guide.webflow.io/

### CODE REVIEW

**Review by lildude — CHANGES_REQUESTED** [review]
>  I have obtained agreement from the wider language community on this color change.
[URL to public discussion]
[Optional: URL to official branding guidelines for the language]

No you haven't 😉

You can't blindly change the language without filling this in. This is strictly enforced as people are sensitive about these things.

See https://github.com/github-linguist/linguist/blob/master/CONTRIBUTING.md#changing-the-color-associated-with-a-language for more details

File: lib/linguist/languages.yml [hunk 1]
```diff
@@ -6819,7 +6819,7 @@ Svelte:
   language_id: 928734530
 Sway:
   type: programming
-  color: "#dea584"
+  color: "#08f48c"
```
Comment (sdankel) [inline lib/linguist/languages.yml:6822]:
```suggestion
  color: "#00F58C"
```

**Comment by sdankel** [comment]
Putting this back in draft as the design team discusses internally. They'll post on this PR when a decision is reached.

**Comment by lildude** [comment]
> Public discussion on this PR

Please don’t. The discussion needs to take place publicly in your community forums and referenced in this PR. Not here.

**Comment by sdankel** [comment]
@lildude I updated the link to our community forum with discussion and a poll. Let me know if this works.

**Review by lildude — APPROVED** [review]
LGTM. Thanks.

Note: this PR will not be merged until close to when the next release is made. See [here](https://github.com/github/linguist/blob/master/docs/troubleshooting.md#my-linguist-pr-has-been-merged-but-github-doesnt-reflect-my-changes) for more details.

**Pull request merged**